### ISSUE TITLE
Should decrypt really fail?

### ISSUE THREAD

**Issue by mbauhardt** [keybase/keybase-issues#3297]
Hi,
I'm using Keybase almost every day in combination with [pass](https://www.passwordstore.org/) as a password manager. See [pass-keybase](https://github.com/mbauhardt/pass-keybase)

Anyway. Before some days I revoked one of my devices, which I used to encrypt my passwords. Now, if I'm using another device to decrypt my passwords I get the error

```
Authored by mbauhardt, however the key that authenticated this message has been revoked (key ID: 01200cec64dad851a5ea3a4c4982b57e492e4f06e7c41444b7b52f9c052b911d5e340a).
Use --force to decrypt anyway.
▶ ERROR Decryption error: For user "mbauhardt": sender key revoked
```

I mean sure, when I use the option `--force` I'm able to decrypt.
The question I have is about: Is It really convenient to not decrypt a message if the sender's key is revoked, or could be a warning message enough?

Thx
Marko

**Comment by 0xdevalias** [comment]
Depending on the usage, I could see this as being a good thing. For example, if I am decrypting in a shell script and piping that data into something else, I would want that to fail if the key is revoked, as that would mean that potentially the key was compromised and I won't want to trust that data anymore.

Seems like a good security choice, that is easily work-aroundable if you actually want to see it.

**Comment by mbauhardt** [comment]
> Seems like a good security choice, that is easily work-aroundable if you actually want to see it.

yes, I think it is fair enough. One use-case would have it in this way the use-case in that way.
